src/components/DpDataTable/DpDataTable.vue

```diff
@@ -484,15 +484,18 @@ export default {
   },
 
   watch: {
-    headerFields () {
-      if (this.isResizable) {
-        this.$nextTick(() => {
-          const firstRow = this.tableEl.getElementsByTagName('tr')[0]
-          const tableHeaderElements = firstRow ? firstRow.children : null
-
-          this.setColsWidth(tableHeaderElements)
-        })
-      }
+    headerFields: {
+      handler () {
+        if (this.isResizable) {
+          this.$nextTick(() => {
+            const firstRow = this.tableEl.getElementsByTagName('tr')[0]
+            const tableHeaderElements = firstRow ? firstRow.children : null
+
+            this.setColsWidth(tableHeaderElements)
+          })
+        }
+      },
+      deep: false // HeaderFields are always replaces as a whole and therefor deep watch is not necessary
```

Comment:
I think this is the default for watchers. See [Vue3 docs](https://vuejs.org/guide/essentials/watchers#deep-watchers).

Reply:
true. I added it, to make clear, that I checked the spot and its fine and the warning won't point me to that line again and again